### ISSUE TITLE
add isOwner to TwitchUser

### ISSUE THREAD

**Issue by RebelliousUno** [Gikkman/Java-Twirk#9]
I'm liking the library, but one thing I've noticed coming into this is that isMod doesn't cover the channel
so I'm having to do 
```
if (sender.isMod || sender.userType == USER_TYPE.OWNER) {
                twirk.channelMessage("Mods Only Message")
}
```

A handy isOwner would be a nice to have on the TwitchUser interface

**Comment by Gikkman** [comment]
Hm, I hadn't thought of it but you are definitely correct. Will add it with the next update

**Comment by Gikkman** [comment]
Also, if you want something that emulates "mod or higher", you can do the following:
```
if (sender.userType.value >= USER_TYPE.MOD.value ) {
    // something
} 
```
( I will still add the isOwner method =)  )

**Comment by RebelliousUno** [comment]
Thanks,  that's useful to know

**Comment by Gikkman** [comment]
This was fixed in #13
